.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -130,23 +130,38 @@ steps:
     displayName: Run VHD cache test
 
 # We can upload release notes for check-in pr and sig mode to validate whether it is expected.
-  - task: PublishPipelineArtifact@0
-    inputs:
-        artifactName: 'vhd-release-notes-${{ parameters.artifactName }}'
-        targetPath: 'release-notes.txt'
+  
+  - bash: |
+      if [[ ${{ parameters.artifactName }} =~ "2019" ]]; then
+        WINDOWS_VERSION="$(cat vhdbuilder/packer/windows-image.env | grep -a "WINDOWS_2019_BASE_IMAGE_VERSION" | cut -d "=" -f 2 | cut -d "." -f 1,2)"
+      elif [[ ${{ parameters.artifactName }} =~ "2022" ]]; then
+        WINDOWS_VERSION="$(cat vhdbuilder/packer/windows-image.env | grep -a "WINDOWS_2022_BASE_IMAGE_VERSION" | cut -d "=" -f 2 | cut -d "." -f 1,2)"
+      else
+        echo "Current distro is not supported to get image version. You need to update related code."
+        exit 1
+      fi
+      IMAGE_VERSION="$WINDOWS_VERSION.${BUILD_DATE}"
```

Comment:
It seems like that `${BUILD_DATE}` is empty
<img width="299" alt="image" src="https://github.com/Azure/AgentBaker/assets/6416187/11bddc59-54a1-4b43-8487-fa7be88e854b">

Reply:
Modify the code for obtaining BUILD_DATE.